lib/views/common/loading_indicator/loading_indicator_utils.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:kayla_flutter_ic/views/common/loading_indicator/loading_indicator_dialog.dart';
+
+extension BuildContextExtension on BuildContext {
```

Comment:
@nmint8m Please rename this file, it should be `BuildContextEdt.dart` and put on `common` folder 🙏